src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -208,6 +217,23 @@ class JooqBurdenEstimateRepository(
         return mapper.mapBurdenEstimateSets(records)
     }
 
+    private fun checkBurdenEstimateSetPathParameters(groupId: String, touchstoneVersionId: String, scenarioId: String,
+                                                     burdenEstimateSetId: Int)
+    {
+        dsl.select(BURDEN_ESTIMATE_SET.ID)
+                .from(BURDEN_ESTIMATE_SET)
+                .join(RESPONSIBILITY)
+                .on(RESPONSIBILITY.ID.eq(BURDEN_ESTIMATE_SET.RESPONSIBILITY))
+                .joinPath(RESPONSIBILITY, SCENARIO, SCENARIO_DESCRIPTION)
+                .joinPath(RESPONSIBILITY, RESPONSIBILITY_SET)
+                .where(BURDEN_ESTIMATE_SET.ID.eq(burdenEstimateSetId))
+                .and(RESPONSIBILITY_SET.MODELLING_GROUP.eq(groupId))
+                .and(RESPONSIBILITY_SET.TOUCHSTONE.eq(touchstoneVersionId))
+                .and(SCENARIO_DESCRIPTION.ID.eq(scenarioId))
+                .fetch()
+                .singleOrNull() ?: throw UnknownObjectError(burdenEstimateSetId, "burden estimate set id")
```

Comment:
Above we use `burden-estimate-set` as the key rather than `burden-estimate-set-id` (and in general we refer to the object rather than the id.) But in fact we should really be using 
```
UnknownObjectError(burdenEstimateSetId, BurdenEstimateSet::class)
```
I know we've been using the string constructor for these errors in loads of places but ideally whenever there's a model class we should be using that rather than constructing our own string, that way we'll avoid consistency issues like this one. Going to try and be more vigilant about this!